safe/sync.go

```diff
@@ -0,0 +1,35 @@
+package safe
+
+import (
+	"fmt"
+)
+
+func (s *Safe) SyncWith(otherSafe *Safe) error {
```

Comment:
See my comment here: https://github.com/bndw/pick/pull/109#discussion_r103090759
It's not just importing, it's syncing existing accounts as well.